modules/nw-dpu-configuring-operator.adoc

```diff
@@ -4,37 +4,42 @@
 
 :_mod-docs-content-type: PROCEDURE
 [id="nw-dpu-configuring-operator_{context}"]
-= Configuring the DPU Operator 
+= Configuring the DPU Operator
 
 To configure the DPU Operator follow these steps:
 
 .Procedure
 
-. Create a `DpuOperatorConfig` custom resource (CR) on both the host cluster and on each of the DPU clusters. The DPU Operator in each cluster is activated after this CR is created.
+. Create the `DpuOperatorConfig` Custom Resource (CR) based on your deployment mode:
 
-. Create a file named `dpu-operator-host-config.yaml` by using the following YAML:
+* Dual Cluster Deployment: You must create the `DpuOperatorConfig` CR on both the host {product-title} cluster and on each of the {ms} DPU clusters.
```

Comment:
This appears in the document as MicroShift (MicroShift) so I'm not sure we need the parenthesis here.

Reply:
Think we need that for the official name "Red Hat build of MicroShift (MicroShift)" that is a variable substitution like {product-title}